app/src/features/detail/components/PackageDetail.tsx

```diff
@@ -46,6 +47,16 @@ interface PackageDetailProps {
   version?: string;
 }
 
+// Adapter for ReactMarkdown's code block signature
```

Comment:
Can we remove this commented section?

Reply:
I had kept that for now to understand, if Syntax highlighting is a must for this PR, should have opened a draft one.
Will remove and also remove a few comments.